From 3d6e62a21f21e57bfa737a21d5c6854fe9365316 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Thu, 11 Aug 2016 13:36:14 -0700
Subject: [PATCH 1723/2848] HADOOP-12465. Incorrect javadoc in
 WritableUtils.java. Contributed by Jagadesh Kiran
 N.

(cherry picked from commit 7fbf69bf47bb733f95b4afd733f9b65e1e7f2b46)
(cherry picked from commit 613c7088fd1accc59fdb7c700ef89409b0fb0d43)
(cherry picked from commit aa0231f921a6cd07d07250dbde12f31701b6fdc6)

Change-Id: I5a7ca9036de53cb035c77f397918a6621ce9bcd7
---
 .../java/org/apache/hadoop/io/WritableUtils.java   |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/WritableUtils.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/WritableUtils.java
index a11a860..6f106f7 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/WritableUtils.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/WritableUtils.java
@@ -236,13 +236,13 @@ public static void cloneInto(Writable dst, Writable src) throws IOException {
 
   /**
    * Serializes an integer to a binary stream with zero-compressed encoding.
-   * For -120 <= i <= 127, only one byte is used with the actual value.
+   * For -112 <= i <= 127, only one byte is used with the actual value.
    * For other values of i, the first byte value indicates whether the
    * integer is positive or negative, and the number of bytes that follow.
+   * If the first byte value v is between -113 and -116, the following integer
+   * is positive, with number of bytes that follow are -(v+112).
    * If the first byte value v is between -121 and -124, the following integer
-   * is positive, with number of bytes that follow are -(v+120).
-   * If the first byte value v is between -125 and -128, the following integer
-   * is negative, with number of bytes that follow are -(v+124). Bytes are
+   * is negative, with number of bytes that follow are -(v+120). Bytes are
    * stored in the high-non-zero-byte-first order.
    *
    * @param stream Binary output stream
-- 
1.7.9.5

